tests/utils/validator_manager.go

```diff
@@ -1246,27 +1272,26 @@ func InitializeAndCompleteEndInitialPoAValidation(
 	ctx context.Context,
 	signatureAggregator *SignatureAggregator,
 	ownerKey *ecdsa.PrivateKey,
-	fundedKey *ecdsa.PrivateKey,
 	l1Info interfaces.L1TestInfo,
 	pChainInfo interfaces.L1TestInfo,
-	stakingManager *poavalidatormanager.PoAValidatorManager,
-	stakingManagerAddress common.Address,
+	validatorManager *validatormanager.ValidatorManager,
+	validatorManagerAddress common.Address,
 	validationID ids.ID,
 	index uint32,
 	weight uint64,
 	pchainWallet pwallet.Wallet,
 	networkID uint32,
 ) {
 	log.Println("Initializing initial validator removal")
-	WaitMinStakeDuration(ctx, l1Info, fundedKey)
+	WaitMinStakeDuration(ctx, l1Info, ownerKey)
 	receipt := InitializeEndPoAValidation(
```

Comment:
I think we should rename all of the `Initialize` util functions to `Initiate` to mirror the rename on the contract side but shouldn't hold up this PR for it.